src/main/resources/application.yml

```
@@ -17,9 +18,18 @@ spring:
 # 애플리케이션 설정
 app:
   batch:
     aimodel-index:
-      enabled: true  # AIModel 인덱싱 스케줄러 활성화 (증분 처리)
\ No newline at end of file
+      enabled: true  # AIModel 인덱싱 스케줄러 활성화 (증분 처리)
+
+# Google API 설정
+google:
+  api:
+    key: ${GOOGLE_API_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)